content/docs/1.8.0/important-notes/_index.md

```diff
@@ -41,6 +42,10 @@ The functionality of the [environment check script](https://github.com/longhorn/
 
 ## General
 
+### Kubernetes Version Requirement
+
+Before upgrading to Longhorn v1.8.0 or a newer version, ensure that all clusters are running Kubernetes v1.25 or later.
```

Comment:
Can we explain the reason behind this? this is because we bump CSI sidecar versions which require the min K8s to be 1.25.